app/queries/integration_mappings/mappings_query.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
 module IntegrationMappings
-  class NetsuiteMappingsQuery < BaseQuery
+  class MappingsQuery < BaseQuery
```

Comment:
What about moving it up one level - similar to integration_integration_items_query:

`class IntegrationMappingsQuery`